gui/src/main/java/io/bisq/gui/util/validation/BsqValidator.java

```diff
@@ -118,4 +125,17 @@ protected ValidationResult validateIfSufficientAvailableBalance(String input) {
             return new ValidationResult(false, Res.get("validation.invalidInput", t.getMessage()));
         }
     }
+
+    protected ValidationResult validateIfMoreThanMinCompensationRequest(String input) {
+        try {
+            final Coin coin = bsqFormatter.parseToCoin(input);
+            if (minCompensationRequest != null && coin.compareTo(minCompensationRequest) <= 0)
+                return new ValidationResult(false, Res.get("dao.compensation.create.amountTooLow",
```

Comment:
We would allow coin == minCompensationRequest, so the <= 0 should be < 0.

Btw in the validateIfSufficientAvailableBalance method above there is wrong comaparison as well:
if (availableBalance != null && coin.compareTo(availableBalance) > 0)
should be the opposite:
if (availableBalance != null && coin.compareTo(availableBalance) < 0)

Reply:
I will refactor the validator to make it more generic with a minAmount. Yes the validateIfSufficientAvailableBalance was right but I will change the order of comparison so its better readable.